ui/components/NotificationCenter/metadata.js

```diff
@@ -105,35 +94,37 @@ const ComponentWithIcon = ({ component }) => {
   const kind = Metadata.toLowerCase();
 
   const paths = [
-    `ui/public/static/img/meshmodels/${modelname}/color/${modelname}-color.svg`,
-    `ui/public/static/img/meshmodels/${modelname}/white/${modelname}-white.svg`,
     `ui/public/static/img/meshmodels/${modelname}/color/${kind}-color.svg`,
     `ui/public/static/img/meshmodels/${modelname}/white/${kind}-white.svg`,
+    `ui/public/static/img/meshmodels/${modelname}/color/${modelname}-color.svg`,
+    `ui/public/static/img/meshmodels/${modelname}/white/${modelname}-white.svg`,
   ];
 
-  const [finalPath, setFinalPath] = useState(
-    'ui/public/static/img/meshmodels/meshery-core/color/meshery-core-color.svg',
-  );
+  const defaultPath = 'ui/public/static/img/meshmodels/meshery-core/color/meshery-core-color.svg';
+
+  const [finalPath, setFinalPath] = useState(defaultPath);
 
   useEffect(() => {
     const loadImages = async () => {
-      if (imageCache[modelname]) {
-        setFinalPath(imageCache[modelname]);
-        return;
+      for (let i = 0; i < 2; i++) {
```

Comment:
Does this loop through SVGs?

Reply:
Yes. You can upload the svg normally and it converts to path so you can directly upload the image.